docs/08-configuration/templates-hierarchy.md

```
@@ -0,0 +1,345 @@
+  - type: "coordinates"
+    bounds:
+      lat: [-23.0, -19.5]  # Limites géographiques NC
+      lon: [163.5, 169.0]
+    crs: "EPSG:3163"  # RGNC91-93
+```
+
+### 2. La Hiérarchie des Templates
+
+Les templates s'organisent en arbre, du plus générique au plus spécifique :
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)